src/zkevm_specs/evm/typing.py

```diff
@@ -215,6 +218,12 @@ def table_fixed(self, randomness: FQ) -> List[TxTableRow]:
                 FQ(0),
                 FQ(1234),  # Mock value for TxSignHash
             ),
+            TxTableRow(
+                FQ(self.id),
+                FQ(TxContextFieldTag.NeutralInvalid),
```

Comment:
What do you think if just calling the table value `IsInvalid`/`Valid`?

Reply:
Sure